src/vivarium_public_health/metrics/mortality.py

```diff
@@ -92,61 +91,44 @@ def columns_required(self) -> Optional[List[str]]:
     def setup(self, builder: Builder) -> None:
         self.clock = builder.time.clock()
         self.config = builder.configuration.stratification.mortality
+        self.causes_of_death = [
```

Comment:
I think this could have stayed in the other method - Im not sure it matters but it made it confusing to read.

Reply:
It's use in two different methods though. I could define it once in `register_observations` and then partial that into `write_mortality_results` but that seems contrived.